.idea/misc.xml

```diff
@@ -29,7 +29,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
I noticed that the languageLevel keeps changing to JDK_1_7 too whenever I work on the app. I take it this isn't something to worry about?